src/BaseModule.php

```diff
@@ -60,20 +58,17 @@ public static function content()
 	 * Extend this method if the module is supposed to process POST requests.
 	 * Doesn't display any content
 	 */
-	public static function post()
+	public static function post(array $parameters = [])
 	{
-		// $a = self::getApp();
```

Comment:
Please don't remove these comments meant to display what `post()` is supposed to end with.

Reply:
Ah, I thought these had been stuff that had been left over.